mpyl-example.py

```diff
@@ -24,9 +29,51 @@ def main(log: Logger, args: argparse.Namespace):
 
     config = parse_config("mpyl_config.yml")
     properties = parse_config("run_properties.yml")
-    run_properties = RunProperties.from_configuration(
-        run_properties=properties, config=config, cli_tag=args.tag
+    cli_parameters = MpylCliParameters(
+        local=args.local,
+        tag=args.tag,
+        pull_main=True,
+        verbose=args.verbose,
+        all=args.all,
     )
+    with Repository(RepoConfig.from_config(config)) as repo:
```

Comment:
Is it possible to centralise this logic of constructing the RunProperties? Because now it's sort of repeated here and also in src/mpyl/build.py and also in src/mpyl/cli/build.py